src/dune_rules/dir_contents.ml

```diff
@@ -302,8 +310,8 @@ end = struct
                    let dirs = [ (dir, [], files) ] in
                    let ml =
                      Memo.lazy_ (fun () ->
-                         let lookup_vlib = lookup_vlib sctx in
                          let loc = loc_of_dune_file st_dir in
+                         let lookup_vlib = lookup_vlib sctx ~loc ~cur_dir:dir in
```

Comment:
`~current_dir` would be preferable please. we don't use `cur` enough that it needs to be shortened.